tests/testthat/test-outputs.R

```diff
@@ -603,3 +603,21 @@ test_that("can generate comparison report with only 1 survey chosen", {
   expect_true(any(grepl("Naomi estimate CY2016Q1", content)))
   expect_true(any(grepl("class=\"logo-naomi\"", content)))
 })
+
+test_that("can generate comparison report without survey ART coverage", {
+  ## Create a model output with only 1 option chosen for survey_prevalence
+  model_output <- a_hintr_output_calibrated$model_output_path
+  output <- qs::qread(model_output)
```

Comment:
Probably use `read_hintr_output` here over `qs::qread` so that way if we change the format of `model_output` again at any point we don't have to update this tests, only that helper function.